src/core/document.js

```diff
@@ -375,6 +397,24 @@ class Page {
     );
   }
 
+  handleDefaultAppearance(partialEvaluator, task) {
+    if (this.defaultAppearanceData) {
+      return new Promise();
+    }
+
+    const opList = new OperatorList();
+
+    const appearanceStream = new StringStream(this.defaultAppearance);
+    this.defaultAppearanceData = Dict.empty;
```

Comment:
We should use `new Dict` here since we should never mutate `Dict.empty` as it's always an empty dictionary (not sure if that is even allowed now, and if so we should probably make that impossible).